fda-model-test/PairedDataTest.cs

```diff
@@ -0,0 +1,29 @@
+using NUnit.Framework;
+using paireddata;
+
+namespace fda_model_test
+{
+    public class PairedDataTest
+    {
+        public static double[] xs = { 0, 2000, 5000, 10000, 25000, 50000 };
+        private static double[] ys = { 556, 562, 565, 566, 570, 575 };
+        PairedData myPairedData = new PairedData(xs, ys);
+
+        [SetUp]
+        public void Setup()
+        {
+ 
+        }
+
+        [Test]
+        public void TestGetXvals()
+        { 
+            Assert.AreEqual(xs,myPairedData.Xvals);
```

Comment:
This is a great test to get something started - but testing access modifiers is probably not super necessary. We will be blue in the face writing this level of testing all the time. We need to test the mission critical code like integrate, multiply and compose. (I am also advocating we seek to protect the xs and ys from public access)

Reply:
I'm good with that. I'd be interested in deleting this right back out with that in mind. I don't want to add noise for no good reason.